consonance/ml_logic/omr.py

```diff
@@ -314,7 +362,9 @@ def decode_batch_predictions(pred):
     # Iterate over the results and get back the text
```

Comment:
I need to repurpose this for the predictions, but tbh I'm still not positive as to what the max_length and the variables it's computed based on (images and labels) represent exactly. Not sure what file name has to do with prediction. Could you please explain?